regexes/client/hints/browsers.yml

```diff
@@ -266,3 +266,28 @@
 'ginxdroid.gdm': 'GinxDroid Browser'
 'on.browser': 'OnBrowser Lite'
 'com.pico.browser.overseas': 'PICO Browser'
+'com.cliqz.browser': 'Cliqz'
+'org.hola.prem': 'hola! Browser'
+'com.baidu.browser.inter': 'Baidu Browser'
+'org.torproject.torbrowser_alpha': 'Tor Browser'
+'com.ilegendsoft.mercury': 'Mercury'
+'com.apn.mobile.browser.coolpad': 'APN Browser'
+'com.apn.mobile.browser.infosonics': 'APN Browser'
+'net.dezor.browser': 'Dezor'
+'com.involta.involtago': 'Involta Go'
+'jp.ddo.pigsty.HabitBrowser': 'Habit Browser'
+'org.browser.owl': 'Owl Browser'
+'com.orbitum.browser': 'Orbitum'
+'com.appsverse.photon': 'Photon'
+'fr.agrange.bbbrowser': 'Keyboard Browser'
+'com.stealthmobile.browser': 'Stealth Browser'
+'com.wcd.talkto': 'TalkTo'
+'com.foxylab.airfox': 'Proxynet'
+'io.github.mthli.goodbrowser': 'Good Browser'
+'app.proxyiumbrowser.android': 'Proxyium'
+'com.vuhuv': 'Vuhuv'
+'com.bbqbar.browser': 'BBQ Browser'
```

Comment:
Are we sure this is a real browser? I was able to find only 2 mentions of this and both suggest, that it may be malware (or at least adware).

Reply:
Yes, it is malware. The BBQ Browser removed.